src/admin-views/widgets/components/checkbox.php

```diff
@@ -11,28 +11,33 @@
  *
  * @version TBD
  *
- * @var string $label Label for the checkbox.
- * @var string $value Value for the checkbox.
- * @var string $id    ID of the checkbox.
- * @var string $name  Name attribute for the checkbox.
+ * @var string $label      Label for the checkbox.
+ * @var string $value      Value for the checkbox.
+ * @var string $id         ID of the checkbox.
+ * @var string $name       Name attribute for the checkbox.
+ * @var string $dependency The dependency attributes for the control wrapper.
  */
 
 ?>
-<p
-		class="tribe-widget-checkbox tribe-common-form-control-checkbox"
+<div
+		class="tribe-widget-form-control tribe-widget-form-control--checkbox"
+		<?php
+		// Not escaped - contains html ( data-attr="value")
+		echo $dependency;
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$dependency'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$dependency'.

<sub>hash: 11392a6e8a0b17a40bb3bd86e0175201</sub>